src/simulator/perftest.py

```diff
@@ -443,15 +446,19 @@ def __init__(self, argv):
         #                     nargs=1,
         #                     help="The path where the result of the run need to be stored.")
 
+        parser.add_argument('-agr', '--skip_auto_gen_report',
```

Comment:
Why not call it skipReport? Then it is consistent with skipDownloads.

Reply:
Good call, thanks @pveentjer, changes applied: https://github.com/hazelcast/hazelcast-simulator/pull/2040/commits/e7d970bbd6b421a159169455ba8949904b828a7b